lib/ui/screens/profile/index.tsx

```
@@ -1,15 +1,88 @@
+                  : 140,
+            shadowOpacity: 0.25,
+            shadowRadius: 4,
+          }}
+        >
+          {isFormOpened === "LICENSE_FORM" && (
+            <DrivingLicenseForm setIsFormOpened={setIsFormOpened} />
+          )}
+          {isFormOpened === "VEHICLE_FORM" && (
+            <VehiclePlateForm setIsFormOpened={setIsFormOpened} />
+          )}
+          {isFormOpened === null && <></>}
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)